src/components/linear-progress/linear-progress.tsx

```diff
@@ -56,6 +56,13 @@ export class LinearProgress {
     }
 
     public render() {
+        /**
+         * Avoid rendering the component when the prop 'value' is explicitly set to null.
+         */
+        if (this.isValueNull(this.value)) {
```

Comment:
What about `undefined`? Or what if it is an `object`? :smile:  
According to the type definition, the prop needs to be a `number`, so it might be better to check for that instead, e.g.
```suggestion
        if (!Number.isFinite(this.value)) {
```

Reply:
I will change the commit message later